lua/entities/gmod_tardis/modules/sh_doors.lua

```diff
@@ -83,6 +83,14 @@ if SERVER then
         end
     end
 
+    ENT:AddHook("HandleE2", "door", function(self,name,e2)
+        if name == "Door" then
+            return self:ToggleDoor()
+        elseif name == "GetDoor" and TARDIS:CheckPP(e2.player, self) then
```

Comment:
We need to move the CheckPP call up to the ToggleDoor if statement rather than this one

Reply:
Should be fixed.